app/views/items/index.html.haml

```diff
@@ -0,0 +1,17 @@
+%h1 Ingredients Matcher
+%p Select some items and we'll give you some recipes
+
+%section.container
+%h4 What's in your fridge?
+- @items.each do |item|
+    %div.item-tile{id: dom_id(item)}
+    %p= item.name
+:Checkbox
```

Comment:
Are you in scope with the checkbox?

Reply:
Sorry this was a test which I forgot to get rid of, I am now scoped